### ISSUE TITLE
The LibGpiodDriver.Linux cannot produce both rising and falling pin events.

### ISSUE THREAD

**Issue by Frankenslag** [dotnet/iot#654]
If I look at AddCallbackForPinValueChangedEvent and WaitForEvent in the LibGpiod driver then I see the following 

```
if ((eventTypes & PinEventTypes.Rising) != 0)
{
      eventHandler.ValueRising += callback;
}
else if ((eventTypes & PinEventTypes.Falling) != 0)
{
      eventHandler.ValueFalling += callback;
}
```

The way I read this is that you could not register for both a rising and falling pin event (you would just get the rising one). Is this intentional ? If not do you want me to create a PR ?

**Comment by krwq** [comment]
yes, looks like a bug. You can go ahead and create PR

**Comment by Frankenslag** [comment]
@krwq
PR #655 created to resolve this.

**Comment by joperezr** [comment]
Fixed by #655
